### PR TITLE
Refactor WS/MQ controllers : use only one listenning topic for ALL controllers

### PR DESCRIPTION
**Pull request by ballinette**
/!\ WARNING - This PR involves refactoring into the SDK implementation, that need to be done before merging  /!\

Use only one listening topic for both every controllers, to simplifiy controllers code maintenance and facilitate the next implementation of plugin controllers.

### CODE REVIEW

File: docs/API.AMQP.md [hunk 1]
```diff
@@ -1413,27 +1295,15 @@ These statistics include:
...
-  clientId: <Unique session ID>,
-
-  /*
-  Optional: Kuzzle will forward this field in its response, allowing you
-  to easily identify which query generated the response you got.
-  */
-  requestId: <Unique query ID>
+  controller: 'admin',
+  action: 'getAllStats',
+  collection: ''
```
Comment (scottinet) [inline docs/API.AMQP.md]: for global route, I think it is best to remove the ``collection`` argument altogether

File: features/support/apiAMQP.js [hunk 2]
```diff
@@ -41,128 +41,155 @@ module.exports = {
...
-    return publish.call(this, topic, msg);
+    return publish.call(this, msg);
   },
 
   globalBulkImport: function (bulk) {
     var
-      topic = ['bulk', '', 'import'].join('.'),
       msg = {
+        controller: 'bulk',
+        collection: '',
```
Comment (scottinet) [inline features/support/apiAMQP.js]: this line should be removed (as with all global routes for all APIs)

**Comment by scottinet** [comment]
+1

File: docs/API.AMQP.md (hunk 1 above)
Comment (ballinette) [inline docs/API.AMQP.md]: Yep, indeed

File: features/support/apiAMQP.js (hunk 2 above)
Comment (ballinette) [inline features/support/apiAMQP.js]: Yep, indeed

**Comment by benoitvidis** [comment]
+1

**Pull request merged**